runtime/src/main/java/com/flipkart/flux/guice/interceptors/TransactionInterceptor.java

```diff
@@ -11,40 +11,45 @@
  * limitations under the License.
  */
 
-package com.flipkart.flux.util;
+package com.flipkart.flux.guice.interceptors;
 
 import org.aopalliance.intercept.MethodInterceptor;
 import org.aopalliance.intercept.MethodInvocation;
+import org.hibernate.HibernateException;
 import org.hibernate.Session;
+import org.hibernate.SessionFactory;
 import org.hibernate.Transaction;
 import org.hibernate.context.internal.ManagedSessionContext;
+import javax.inject.Inject;
 
 /**
- * Provides transactional boundaries to methods which are annotated with {@link com.flipkart.flux.util.Transactional}.
+ * Provides transactional boundaries to methods which are annotated with {@link javax.transaction.Transactional}.
  * @author shyam.akirala
  */
 public class TransactionInterceptor implements MethodInterceptor {
```

Comment:
As runtime highly depends on this, would like to keep it here itself, also
persistence-mysql doesn't have to depend on Guice libraries which are
unnecessary to it.

On Tue, Apr 12, 2016 at 9:59 AM, Kartik Bommepally <notifications@github.com
> wrote:

> In
> runtime/src/main/java/com/flipkart/flux/guice/interceptors/TransactionInterceptor.java
> <https://github.com/flipkart-incubator/flux/pull/3#discussion_r59319430>:
>
> >
> >  /**
> > - * Provides transactional boundaries to methods which are annotated with {@link com.flipkart.flux.util.Transactional}.
> > + * Provides transactional boundaries to methods which are annotated with {@link javax.transaction.Transactional}.
> >   * @author shyam.akirala
> >   */
> >  public class TransactionInterceptor implements MethodInterceptor {
>
> Isn;t persistence-mysql module a better place for such generic classes
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/flipkart-incubator/flux/pull/3/files/b25f3139f078dc56608d666b4e268feba2a516d4#r59319430>
>



-- 
- Shyam